src/main/proto/com/kodypay/grpc/pay/v1/pay.proto

```diff
@@ -92,3 +93,120 @@ message RefundResponse {
     REFUNDED = 3;
   }
 }
+
+// requires X-API-Key header with 'API Key' value
+message AuthorizationRequest {
```

Comment:
@endrec Can I understand the reason why this is needed?
What would happen in the backend for this?

Reply:
In short, it's the first half of a standard to step payment (https://kodypay.atlassian.net/browse/PBBV0-189), so the backend should ask the terminal to authorise a transaction up to `amount`, and return the relevant details so at a later point we can capture from this authorization, or release it.
Bear in mind that Symphony uses this same transaction type to check the validity of a card, where `amount` is `0`.

Use case: you walk into the bar, and ask the bartender to open you tab with a £100.

Reply:
So this is a `PreAuth` request?
I get that now.
A follow up question: are all of these parameters required?  They seem specific for the needs of Oracle instead parameters that fit into our data model

Reply:
> are all of these parameters required? They seem specific for the needs of Oracle instead parameters that fit into our data model

Some of them is optional (as marked), and I tried to strip out what I thought is really oracle specific, the ones here I believe could be useful for any integrator.